server/app/services/export/PdfExporter.java

```diff
@@ -84,8 +98,21 @@ private byte[] buildPDF(
             new Paragraph(
                 answerData.questionDefinition().getName(),
                 FontFactory.getFont(FontFactory.HELVETICA_BOLD, 12));
-        Paragraph answer =
-            new Paragraph(answerData.answerText(), FontFactory.getFont(FontFactory.HELVETICA, 11));
+        Paragraph answer = null;
+        if (answerData.fileKey().isPresent()) {
+          String encodedFileKey =
```

Comment:
This doesn't appear to be using the helper methods from `FileUploadQuestion`. Did that approach not work?

Reply:
Which helper method? I don't see any method that gets a encodedFilekey

Reply:
I'd misunderstood. Not blocking this PR, but it might make sense to change `AnswerData.fileKey` to be `AnswerData.encodedFileKey()` and do the encoding in the place that sets it since all consumers of it turn right around and encode it.